pkg/cli/newapp/newapp.go

```diff
@@ -134,9 +134,16 @@ To list all local templates and image streams, use:
 To search templates, image streams, and Docker images that match the arguments provided, use:
 
   oc new-app -S php
-  oc new-app -S --template=ruby
+  oc new-app -S --template=rails
   oc new-app -S --image-stream=mysql
+  # REMINDER, for docker image references, if a registry or repository are not specified, 'docker.io' and 'library' are potential defaults for the various image registry clients
   oc new-app -S --docker-image=python
+
```

Comment:
> This is for oc new-app's search capability. 

Yes that is correct.

> Will this only search Docker Hub images? Are quay.io images also included in the search results?

It is a function of the imagestreamimport API actually, that is what is used from oc new-app.  It will search other registries but as it turns out with docker.io having precedence with the core docker clients used, the only image with name of "python" and nothing else is "docker.io/library/python" so that is the hit that is obtained.  There is not quay.io/library/python for example.

In addition to the warning I could have also changed the `oc new-app -S --docker-image=python` to `oc new-app -S --docker-image=<whatever python images exists on quay.io>` but that looked clunky to me.

But if you want to go that approach @adambkaplan give an explicit go ahead and we'll go from there.